src/main/resources/bidder-config/nextmillenium.yaml

```diff
@@ -0,0 +1,17 @@
+adapters:
+  nextmillenium:
+    endpoint: https://pbs.nextmillmedia.com/openrtb2/auction
+    meta-info:
+      maintainer-email: appdevs@nextmillennium.io
+      app-media-types:
+        - banner
+      site-media-types:
+        - banner
+      supported-vendors:
+      vendor-id: 0
```

Comment:
Should be `1060`
As listed in https://vendor-list.consensu.org/v2/archives/vendor-list-v118.json